superset/assets/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -108,6 +109,34 @@ class HeaderActionsDropdown extends React.PureComponent {
     this.props.startPeriodicRender(refreshInterval * 1000);
   }
 
+  downloadChartImage(ev){
+    //debugger;
+    var jtarget = $(ev.currentTarget)
```

Comment:
in general these variables are a bit vaguely named. In general, using full words for variable names is preferred, so something like `backgroundColor` and `backgroundColors` below. i'd recommend something like `dashboardElement` here.